src/tests.rs

```diff
@@ -260,3 +260,27 @@ fn test_run_main_without_main() {
     ));
     run_main(module, 100_000, &*interface).expect_err("An error should spawn here");
 }
+
+#[test]
+#[serial]
+fn test_run_empty_main() {
+    settings::reset_metering();
+    let interface: Box<dyn Interface> =
+        Box::new(TestInterface(Arc::new(Mutex::new(Ledger::new()))));
+    let module = include_bytes!(concat!(
+        env!("CARGO_MANIFEST_DIR"),
+        "/wasm/build/empty_main.wasm"
+    ));
```

Comment:
This will fail to compile because you didn't push the `.wasm` file, only the TS code. Not sure it's the best solution to push the built files but it does save a running step when running the tests and it is also what has been done so far.